config/app.yaml.ctmpl

```diff
@@ -4,7 +4,7 @@
 {{$study_guid := env "STUDY_GUID"}}
 
 service: {{$study}}
-runtime: python37
+runtime: python312
```

Comment:
Based off app.yaml examples in the Google reference page, we can use runtime python312.

Replaced `python37` with `python312` everywhere in this project.